### PR TITLE
Societyworks demo cobrand (including Waste demo)

### PR DESCRIPTION
**Pull request by zarino**
**To do**

- [x] Reband Borsetshire cobrand as SocietyWorks Showcase
- [x] Implement existing `/waste` features
- [x] Add (stubbed-out) waste features:
   - [ ] Sign up for SMS/email reminders before collection days
   - [ ] Pay for bulky container pickup (with a faked third-party payment screen)
   - [ ] Requesting new waste services at a property (eg: new-build not getting recycling collection yet)
- [x] Improve styling for `/waste` features (less GOV.UK, more harmonised with SocietyWorks / FMS form styles)
- [x] Decide what to do about the bottom half of the homepage and the waste index page
- [x] Decide whether / how the cobrand should link to societyworks.org
- [x] ~~Check whether there are any other recent features that need enabling in this cobrand (eg: assets) and whether they require any front-end changes~~ – being tracked in https://github.com/mysociety/fixmystreet-commercial/issues/2009

home

reporting

waste

### CODE REVIEW

**Comment by dracos** [comment]
"time to switch from Google Maps" - demo site uses OSM.

**Comment by zarino** [comment]
> "time to switch from Google Maps" - demo site uses OSM.

Oh, so it does! I saw [google_maps_style.js](https://github.com/mysociety/fixmystreet/blob/e2fd716/web/cobrands/borsetshire/google_maps_style.js) in the borsetshire assets directory, and assumed it must still be using Google Maps… but, maybe that file isn’t even needed any more?

**Comment by LouiseMySociety** [comment]
Spoke to Zarino about progress and what's needed next. Made this related ticket https://github.com/mysociety/fixmystreet-commercial/issues/2009

File: templates/web/base/alert/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@ <h1>[% loc('Local RSS feeds and email alerts') %]</h1>
       [% INCLUDE 'around/_postcode_submit_button.html' attr='class="green-btn"' %]
     </div>
   </fieldset>
+  [% INCLUDE 'around/_postcode_form_geolocation.html' url='/alert/list' %]
   <a href="/alert/list" id="geolocate_link">&hellip; [% loc('or use my location') %]</a>
```
Comment (dracos) [inline templates/web/base/alert/index.html:19]: Should this line be removed then?

File: templates/web/borsetshire/around/postcode_form.html [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-        [%
-            question = c.cobrand.enter_postcode_text || loc('Enter a nearby street name and area');
-        %]
-
-        <form action="[% c.uri_for('/around') %]" method="get" name="postcodeForm" id="postcodeForm">
-            <label for="pc">[% question %]:</label>
-            <p class="form-hint" id="pc-hint">[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]</p>
-            <div>
-                <input type="hidden" name="pc" value="[% c.cobrand.problems.first.postcode | html %]">
-                <input type="text" name="_pc" value="[% pc | html %]" id="pc" size="10" maxlength="200" aria-describedby="pc-hint" required>
```
Comment (dracos) [inline templates/web/borsetshire/around/postcode_form.html:23]: This functionality of a hidden pc and _pc for what they enter (so it always takes them to the pins) has been lost, could we have it back?

File: perllib/FixMyStreet/Cobrand/Borsetshire.pm [hunk 3]
```diff
@@ -27,4 +32,141 @@ sub send_questionnaires {
...
+    my $data = [ map { {
+        value => $_->{uprn},
+        label => $_->{address}
+    } } sort { $a->{address} cmp $b->{address} } values %$example_properties ];
+    return $data;
+}
+
+sub look_up_property {
+    my $self = shift;
+    my $uprn = shift;
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Borsetshire.pm:115]: Would be good to have some tests for all this, I can add some basic ones.

**Review by dracos — CHANGES_REQUESTED** [review]
Looks good, couple of issues.

**Comment by dracos** [comment]
Going to deploy on this branch for now, and won't be merged into that branch now after tidying, so closing this but is deployed.

**Pull request closed without merging**